internal/lib/checkpoint.go

```diff
@@ -172,26 +172,42 @@ func (c *ContainerServer) ContainerCheckpoint(ctx context.Context, opts *Contain
 // Copied from libpod/diff.go
 var containerMounts = map[string]bool{
 	"/dev":               true,
-	"/etc/hostname":      true,
-	"/etc/hosts":         true,
-	"/etc/resolv.conf":   true,
+	"/dev/shm":           true,
```

Comment:
In Podman was introduced support for checkpoint/restore for the content of `/dev/shm` (https://github.com/containers/podman/pull/12665).
Do we need similar functionality in cri-o as well?

Reply:
Working on it in a follow-up PR.